neqo-transport/tests/server.rs

```diff
@@ -939,3 +941,76 @@ fn closed() {
     let res = server.process(None, now() + Duration::from_secs(60));
     assert_eq!(res, Output::None);
 }
+
+#[test]
+fn max_streams() {
+    const MAX_STREAMS: u64 = 100;
```

Comment:
You can probably use a smaller value safely.

```suggestion
    const MAX_STREAMS: u64 = 40;
```